egs2/TEMPLATE/asr2/README.md

```diff
@@ -140,30 +142,57 @@ See also:
 
 Upload the trained model to Hugging Face for sharing. Additional information at [Docs](https://espnet.github.io/espnet/espnet2_tutorial.html#packing-and-sharing-your-trained-model).
 
-## How to run
-
-### LibriSpeech Training
-Here, we show the procedure to run the recipe using `egs2/librispeech/asr2`.
-
-Move on the recipe directory.
-```sh
-$ cd egs2/librispeech/asr2
+## How to create asr2 recipes from asr1
 ```
-
-Modify `LIBRISPEECH` variable in `db.sh` if you want to change the download directory.
-```sh
-$ vim db.sh
-```
-
-Modify `cmd.sh` and `conf/*.conf` if you want to use job scheduler.
-See the detail in [using job scheduling system](https://espnet.github.io/espnet/parallelization.html).
-```sh
-$ vim cmd.sh
+1. mkdir egs2/<recipe name>/asr2
+2. cd egs2/<recipe name>/asr2
+3. ../../TEMPLATE/asr2/setup.sh . # create the default files
+4. cd local
+5. ln -s ../../asr1/local/* .     # add symlinks for data preparation files in asr1 except for path.sh
+6. cd ../
+7. cp ../../librispeech/asr2/run.sh .         # copy run.sh
+8. cp ../../librispeech/asr2/conf/tuning/train_discrete_asr_e_branchformer1.yaml conf/ # copy training conf
+9. cp ../../librispeech/asr2/conf/decode_ctc0.3.yaml conf/     # copy confs
+10. EDIT run.sh by checking ../asr1/run.sh
+  a. We may skip an LM
+  b. change language-relatedd setting (e.g., tgt_lang, tgt_nbpe (e.g., change to char for CKJ))
+  c. training, dev, and test sets
+  d. config files
 ```
 
-Run `run.sh`, which conducts all of the stages explained above.
-```sh
-$ ./run.sh
+## Tips for asr2 experiments
+* **Kmeans is important**
+   * SSL model choice can affect the performance a lot, e.g. wavlm models may not work for non-English data,
+   * Layer selection is also important: different layers retain different information. For example, based the training criterion, the 24-th layer of HuBERT_large is trying to match the information from HuBERT_base layer 9. If you didn't have experience, please refer to the Fig. 4 of this [CCA paper](https://arxiv.org/pdf/2211.03929.pdf), which is usually helpful.
+   * Number of kmeans clusters also affect the variance in pronunciation, etc.
+   * Please check the kmeans labels in `dump/extracted/{kmeans_feat_type}/layer{layer}/{dset}/pseudo_label_km{ncluseters}.txt`. In my experience, a good km result for ASR should have an obvious pattern of repeatitions, e.g.
```

Comment:
Maybe, you can add a bad example as well?

Reply:
Sure.